scalatest/src/main/scala/org/scalatest/exceptions/StackDepth.scala

```diff
@@ -90,7 +90,7 @@ trait StackDepth { this: Throwable =>
   }
 
   private def stackTraceElement: Option[StackTraceElement] = {
-    val stackTrace = getStackTrace()
+    val stackTrace = getStackTrace().map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.